strands_movebase/launch/obstacles.launch

```diff
@@ -66,12 +66,6 @@
         <param name="cutoff_z" value="0.3"/>
     </node>
 
-    <!-- This node removes the desired cutoff pixels from the cloud edges -->
```

Comment:
because we want it running even when we don't use the chest cam. ``obstacles.launch`` is only launched when ``with_chest_camera=true``.

Before it wouldnt make a difference because we would load the stuff from scitos_2d_nav instead, but now I think it does.